### PR TITLE
Remote execution to 1.9 (DEB)

### PR DESCRIPTION
**Pull request by stbenjam**
You need this to make REX available for 1.9, right?

### CODE REVIEW

**Comment by mmoll** [comment]
only 0.0.1? :)
At least the dependencies of dynflow are missing.

**Comment by stbenjam** [comment]
I didn't look closely, it was scripted but guess it didn't grab all my commits, will fix

File: dependencies/trusty/dynflow/changelog [hunk 1]
```diff
@@ -0,0 +1,5 @@
+ruby-dynflow (0.8.5-1) stable; urgency=low
```
Comment (iNecas) [inline dependencies/trusty/dynflow/changelog]: 0.8.7 is the current one

**Comment by stbenjam** [comment]
I think we can just drop this, 1.10 is almost released, and next iteration of REX is 1.10 or higher

**Pull request closed without merging**